src/pyhf/tensor/jax_backend.py

```diff
@@ -42,6 +42,8 @@ class jax_backend(object):
 
     def __init__(self, **kwargs):
         self.name = 'jax'
+        self.float_precision = '64b'
+        self.int_precision = '64b'
```

Comment:
do we want to take this opportunity to enable 32-bit jax? Or leave for later?

Reply:
leave for later